tldw_Server_API/tests/Admin/test_admin_smoke.py

```
@@ -57,6 +60,11 @@ def _fresh_client(test_mode: bool = False, user_db_base_dir: Optional[str] = Non
     return client
 
 
+def _managed_sqlite_backend_targets() -> set[str]:
+    registry = getattr(backend_factory, "_sqlite_backend_registry", {})
+    return {str(signature[1]) for signature in registry.keys()}
```

Ruff (0.15.11)

[warning] Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)